# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1523298801 25200
#      Mon Apr 09 11:33:21 2018 -0700
# Branch stable
# Node ID 1b1b29705d532372e7b7cd17d9fc179ce1b9720f
# Parent  e3d5c1def2c8690972beb7411357b56c4a7793ed
no need to set page_screen_output option with --traditional

* interpreter.cc (interpreter::maximum_braindamage): Remove page_screen_output
from the list of options set by --traditional, it is disabled by default now.
* basics.txi: Remove page_screen_output from the list of options associated
with the --traditional option.

diff --git a/doc/interpreter/basics.txi b/doc/interpreter/basics.txi
--- a/doc/interpreter/basics.txi
+++ b/doc/interpreter/basics.txi
@@ -249,17 +249,16 @@ PS2                             = ""
 beep_on_error                   = true
 confirm_recursive_rmdir         = false
 crash_dumps_octave_core         = false
 disable_diagonal_matrix         = true
 disable_permutation_matrix      = true
 disable_range                   = true
 fixed_point_format              = true
 history_timestamp_format_string = "%%-- %D %I:%M %p --%%"
-page_screen_output              = false
 print_empty_dimensions          = false
 save_default_options            = "-mat-binary"
 struct_levels_to_print          = 0
 @end group
 @end example
 
 @noindent
 and disable the following warnings
diff --git a/libinterp/corefcn/interpreter.cc b/libinterp/corefcn/interpreter.cc
--- a/libinterp/corefcn/interpreter.cc
+++ b/libinterp/corefcn/interpreter.cc
@@ -1293,17 +1293,16 @@ namespace octave
     Fbeep_on_error (octave_value (true));
     Fconfirm_recursive_rmdir (octave_value (false));
     Fcrash_dumps_octave_core (octave_value (false));
     Fdisable_diagonal_matrix (octave_value (true));
     Fdisable_permutation_matrix (octave_value (true));
     Fdisable_range (octave_value (true));
     Ffixed_point_format (octave_value (true));
     Fhistory_timestamp_format_string (octave_value ("%%-- %D %I:%M %p --%%"));
-    Fpage_screen_output (octave_value (false));
     Fprint_empty_dimensions (octave_value (false));
     Fsave_default_options (octave_value ("-mat-binary"));
     Fstruct_levels_to_print (octave_value (0));
 
     disable_warning ("Octave:abbreviated-property-match");
     disable_warning ("Octave:data-file-in-path");
     disable_warning ("Octave:function-name-clash");
     disable_warning ("Octave:possible-matlab-short-circuit-operator");
